### PR TITLE
Design tweaks and JSON-safe escaping string

### PR DESCRIPTION
**Pull request by diamondburned**
A lot of people have been getting a "Cannot send an empty message", as their hostname contains characters that weren't JSON-safe (as in you can't literally just throw the variable into a pre-made JSON body and ship it away). 

Since `jq` was made into a forced dependency (which I'm not happy about, since only one game u-), I've decided to put it to good use.

### CODE REVIEW

**Comment by dgibbs64** [comment]
Cool. We are finding new uses for jq. Used it to display if a server is on the steam master server as well.

File: lgsm/functions/alert_discord.sh [hunk 1]
```diff
@@ -5,60 +5,65 @@
 # Website: https://linuxgsm.com
 # Description: Sends Discord alert.
 
+if ! command -v jq > /dev/null; then
+    fn_print_fail_nl "Sending Discord alert: jq is missing."
+    fn_script_log_fatal "Sending Discord alert: jq is missing."
+fi
+
+EscapedServername="$(echo -n "${servername}" | jq -sRr "@json")"
```
Comment (Bourne-ID) [inline lgsm/functions/alert_discord.sh]: Do we use camel case for variable names elsewhere in the project?

Comment (diamondburned) [reply lgsm/functions/alert_discord.sh]: Nope, just my habit. I'll push a commit tomorrow if I have time as it's late now.

**Comment by diamondburned** [comment]
Made a commit to fit Camel Case requirements + removed extra debug lines.

**Comment by diamondburned** [comment]
Changed, although I'm not sure if it'll be JSON-friendly or not (`jq` might have that handled though)

**Pull request merged**